### PR TITLE
Add table oci_bds_bds_instance

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
> select * from oci_bds_bds_instance where id = 'ocid1.bigdataservice.oc1.ap-mumbai-1.amaaaaaapvq3y5aadwjlwenopfc7jpmc3zig5t5choaz4ip5x3a7deao6hla'
+---------------------------------------------------------------------------------------------------+--------------+-----------------+---------------------------+-----------------+----------------------+>
| id                                                                                                | display_name | cluster_profile | time_created              | lifecycle_state | is_high_availability |>
+---------------------------------------------------------------------------------------------------+--------------+-----------------+---------------------------+-----------------+----------------------+>
| ocid1.bigdataservice.oc1.ap-mumbai-1.amaaaaaapvq3y5aadwjlwenopfc7jpmc3zig5t5choaz4ip5x3a7deao6hla | test43       | HADOOP_EXTENDED | 2023-06-06T13:46:52+05:30 | CREATING        | false                |>
|                                                                                                   |              |                 |                           |                 |                      |>
|                                                                                                   |              |                 |                           |                 |                      |>
|                                                                                                   |              |                 |                           |                 |                      |>
+---------------------------------------------------------------------------------------------------+--------------+-----------------+---------------------------+-----------------+----------------------+>

```
</details>

### CODE REVIEW

**Comment by ParthaI** [comment]
Reference: https://github.com/turbot/steampipe-plugin-oci/pull/455

File: oci/table_oci_big_data_service_instance.go [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+	if err != nil {
+		logger.Error("oci_big_data_service_instance.getBigDataServiceInstance", "api_error", err)
+		return nil, err
+	}
+	return response.BdsInstance, nil
+}
+
+//// TRANSFORM FUNCTIONS
+
+func bdsBdsInstanceTags(_ context.Context, d *transform.TransformData) (interface{}, error) {
```
Comment (madhushreeray30) [inline oci/table_oci_big_data_service_instance.go:284]:
```suggestion
func bigDataServiceInstanceTags(_ context.Context, d *transform.TransformData) (interface{}, error) {
```

**Review by madhushreeray30 — COMMENTED** [review]
@ParthaI can you please have a look at the comments, thanks!

**Review by madhushreeray30 — APPROVED** [review]
LGTM 👍

**Comment by misraved** [comment]
@ParthaI, please update the table name to use `oci_bds_bds_instance`, per terraform docs - https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/bds_bds_instance

**Review by misraved — APPROVED** [review]

**Pull request merged**